requirements.txt

```diff
@@ -1,9 +1,9 @@
-psutil==5.8.0
+psutil~=5.8.0
 requests~=2.27.1
-grpcio==1.43.0
-protobuf==3.19.4
+grpcio~=1.43.0
```

Comment:
Mmm maybe should have left grpcio pinned as its build time is substantial?

Reply:
Users responsible for pinning if needed.